app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -356,6 +356,45 @@ def test_saves_branches_single_feature(self):
         self.assertTrue(treatment_feature_value.enabled)
         self.assertEqual(treatment_feature_value.value, "{'this': 'that'}")
 
+    def test_saves_prevent_pref_conflict(self):
```

Comment:
You could just fold this into an existing saving test couldn't you?

Also I try to avoid putting these kinds of multiple steps, multiple cases into a single test, because if it fails you're not sure which of the cases is failing without inspecting further, so if we kept this I would split it into multiple tests, but even then it would kind of be testing the same thing twice.  All we need to know is that it'll update the db for some new value, what the value is kind of doesn't matter since that's all enforced by the serializer anyways.